notebooks/llm-agent-functioncall/llm-agent-functioncall-qwen.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    model_id = "Qwen/Qwen2-7B-Instruct"
</pre><p>could you please use smaller qwen version (0.5b or 1.5b) for patching in tests</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Yes, I have tested 1.5b, but its result is not good enough for multiple-round function calling

Reply:
I mean only for patching, github action runners does not have enough RAM for conversion and running 7b models, that is why I'm asking about some light-weight model for running tests only (will help maintain common notebook functional  health and simplify debugging if something changed in API)